commcare_export/data_types.py

```diff
@@ -5,12 +5,15 @@
 DATA_TYPE_DATE = 'date'
 DATA_TYPE_DATETIME = 'datetime'
 DATA_TYPE_INTEGER = 'integer'
+DATA_TYPE_NUMERIC = 'numeric'
 
 DATA_TYPES_TO_SQLALCHEMY_TYPES = {
+    DATA_TYPE_TEXT: sqlalchemy.Text(),
     DATA_TYPE_BOOLEAN: sqlalchemy.Boolean(),
     DATA_TYPE_DATETIME: sqlalchemy.DateTime(),
     DATA_TYPE_DATE: sqlalchemy.Date(),
     DATA_TYPE_INTEGER: sqlalchemy.Integer(),
+    DATA_TYPE_NUMERIC: sqlalchemy.Numeric(),
```

Comment:
What if you use `DECIMAL`?

Reply:
Oh, good idea, I'll try that

Reply:
hmm, still looks like it's getting rounded. Will do a little more digging when I get time to set up MS or MySQL